cooperative_lanechange/src/cooperative_lanechange.cpp

```diff
@@ -148,15 +147,25 @@ namespace cooperative_lanechange
         lanelet::ConstLanelet start_lanelet = veh2_lanelet;
         lanelet::ConstLanelet end_lanelet = current_lanelet;
         lanelet::traffic_rules::TrafficRulesUPtr traffic_rules = lanelet::traffic_rules::TrafficRulesFactory::create(lanelet::Locations::Germany,lanelet::Participants::VehicleCar);
-        ROS_DEBUG_STREAM("traffic rules created]");
+        ROS_DEBUG_STREAM("traffic rules created");
         
         lanelet::routing::RoutingGraphUPtr map_graph = lanelet::routing::RoutingGraph::build(*(wm_->getMap()), *traffic_rules);
```

Comment:
You should use wm.getMapRoutingGraph() here instead of building it yourself which is very time consuming